BEP191.md

```diff
@@ -0,0 +1,204 @@
+<pre>
+  BEP: 191
+  Title: Validator Slash Mechanism 2.0
```

Comment:
`Validator Slash Mechanism 2.0` is not a preferred title, any possibility to hight light the key word of the proposal in the title?